ols/utils/query_filter.py

```diff
@@ -0,0 +1,42 @@
+"""A class helps redact the question based on the regex filters provided in the config file."""
+
+import logging
+import re
+from collections import namedtuple
+
+from ols.utils import config
+
+logger = logging.getLogger(__name__)
+
+RegexFilter = namedtuple("RegexFilter", "pattern, name, replace_with")
+
+
+class QueryFilter:
```

Comment:
Nit: What we call a QueryFilter in config is a RegexFilter here.

Reply:
@syedriko I think this is okay, thats the name of the namedtuple. Would leave as it is.